peachjam/templates/peachjam/provision_enrichment/_uncommenced_provision_detail.html

```diff
@@ -2,13 +2,17 @@
 <h6 class="mb-2 provision-toggle">
   <button type="button"
           class="btn btn-sm bg-none {% if not expanded %}collapsed{% endif %}"
+          hx-get="{% url 'uncommenced_provision_detail' enrichment.pk %}"
+          hx-target="#provision-detail-{{ enrichment.pk }}"
+          hx-swap="innerHTML"
+          hx-trigger="click once"
           data-bs-toggle="collapse"
           data-bs-target="#provision-detail-{{ enrichment.pk }}"
           aria-expanded="{% if expanded %}true{% else %}false{% endif %}"
           aria-controls="provision-detail-{{ enrichment.pk }}">
     <i class="bi"></i>
   </button>
-  <a href="{{ enrichment.document.get_absolute_url }}#{{ enrichment.provision_eid }}"
+  <a href="{{ document_url }}#{{ enrichment.provision_eid }}"
      class="provision-link me-2">{{ enrichment.provision_title }}</a>
```

Comment:
`enrichment.provision_title` is increasing the load time locally from 18 queries in 99ms to 34 queries in 175ms